src/CMakeLists.txt

```diff
@@ -214,6 +214,10 @@ ELSE()
   ENDIF()
 ENDIF()
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Well... we'll add a better PPA. We have the technology!

(_and if not I'll add the workaround, ok_)